CMakeLists.txt

```diff
@@ -373,7 +373,7 @@ OCV_OPTION(WITH_XINE "Include Xine support (GPL)" OFF
 OCV_OPTION(WITH_CLP "Include Clp support (EPL)" OFF
   VISIBLE_IF TRUE
   VERIFY HAVE_CLP)
-OCV_OPTION(WITH_OPENCL "Include OpenCL Runtime support" (NOT ANDROID AND NOT CV_DISABLE_OPTIMIZATION)
+OCV_OPTION(WITH_OPENCL "Include OpenCL Runtime support" (NOT CV_DISABLE_OPTIMIZATION)
```

Comment:
`self.opencl` check in build_sdk.py doesn't make sense after this change (it is already enabled).